config/katello-answers.yaml

```diff
@@ -1,41 +1,54 @@
+# Format:
+# <classname>: false - don't include this class
+# <classname>: true - include and use the defaults
+# <classname>:
+#   <param>: <value> - include and override the default(s)
+#
+# See params.pp in each class for what options are available
+
 ---
 certs:
   group: foreman
 katello: true
+global: true
+application: true
```

Comment:
The global and application naming conventions will be confusing to users and be incongruent with a base Foreman install. Could these two be combined into just the katello options from the users perspective?

Reply:
Once we've dropped Pulp 2, it may be time to implement `foreman::plugin::katello`. Not entirely sure yet, but we should take a look.